tests/restApi/test_get_request.py

```diff
@@ -0,0 +1,45 @@
+import requests
+import pytest
+
+
+BASE_URL = "https://drb-api-qa.nypl.org"
+url = BASE_URL + "/collection/3650664c-c8be-4d07-8d64-2d7003b02048"
+
+def test_get_request():
+    response = requests.get(url)
+    assert response.status_code == 200, f"Expected status code 200, but got {response.status_code}"
+
+def test_collection_name():
+
+    response = requests.get(url)
+    assert response.status_code == 200, f"Expected status code 200, but got {response.status_code}"
+
+    # Parse the JSON response
```

Comment:
are these comments necessary?

Reply:
can remove. I put it for my own readability